config/openapi/vendor-api-v1.1.yml

```diff
@@ -1604,6 +1609,32 @@ components:
           description: The time this update was made
           format: date-time
           example: 2019-06-13T23:59:59Z
+    Pagination:
+      type: object
+      description: pagination
+      additionalProperties: false
+      required:
+      - first
+      - last
+      - prev
+      - next
+      properties:
+        first:
+          type: string
+          description: The first page of the results
+          example: https://www.apply-for-teacher-training.service.gov.uk/api/v1.1/applications?since=2021-10-20T12:00:00Z?page=1
+        last:
+          type: string
+          description: The last page of the results
+          example: https://www.apply-for-teacher-training.service.gov.uk/api/v1.1/applications?since=2021-10-20T12:00:00Z?page=20
+        prev:
+          type: string
+          description: The previous page of the results
+          example: https://www.apply-for-teacher-training.service.gov.uk/api/v1.1/applications?since=2021-10-20T12:00:00Z?page=1
```

Comment:
looks a bit off in the example response `prev` being the same as `first` rather than `null`:
![image](https://user-images.githubusercontent.com/47917431/144104802-5a00e905-ee40-4b2d-a83b-674cd9998c16.png)
but null wouldn't be an accurate representation of the example?

Reply:
For this example, we're pretending we are on page 2. I've modified the links accordingly.